stripe/createCheckoutSession.js

```diff
@@ -1,22 +1,51 @@
 import Stripe from 'stripe';
+import AWS from 'aws-sdk';
+import Joi from '@hapi/joi';
 
-import { failure, resourceNotFound, success } from '../libs/response-lib';
-import * as dynamoDbLib from '../libs/dynamodb-lib';
+import { failure, resourceNotFound, success, validationError } from '../libs/response-lib';
 import * as secretManagerLib from '../libs/secretmanager-lib';
 import { getEnvironment } from '../libs/utils-lib';
 // eslint-disable-next-line no-unused-vars
 import typings from '../typings/stripeSecrets';
 
+/**
+ * @typedef {Object} CheckoutSessionRequestParams
+ * @property {number} amount
+ * @property {String} companyId
+ * @property {String} customerEmail
+ */
+
+const couponCheckoutSchema = Joi.object({
+  amount: Joi.number()
+    .min(1000)
+    .max(25000)
+    .required(),
+  companyId: Joi.string()
+    .guid()
+    .required(),
+  customerEmail: Joi.string()
+    .email()
+    .required(),
+});
+
```

Comment:
This can be extracted in a separate file. There is a folder for this `validation`

Reply:
this one will never be reused outside of this file. I feel like it will be more convenient to see the validation rules right next to the only place where these rules are used.

Reply:
I know. But even `companySchema` is used in one place. I think this is unnecessary to be in this file. We should aim for separation of concern as much as possible.